webapp/sources/rudder/rudder-web/src/main/elm/sources/Editor/ViewTechniqueTabs.elm

```diff
@@ -62,26 +63,38 @@ techniqueParameter model technique param opened =
         ( "Parameter cannot be empty and needs a value.\nIf you add a parameter to an existing technique, policy generation will fail and you will need to update all directives with the new parameter value"
         , "btn-info"
         )
+    (invalidParamClass, invalidParamElem) =
+        if (Regex.contains ((Regex.fromString >> Maybe.withDefault Regex.never) "[^_a-zA-Z\\d]") param.name) then
+          ("error",  ul [class "row"] [ li [ class "text-danger col-sm-8" ] [ text "Invalid variable name is required, valid characters are [_a-zA-Z\\d]" ] ] )
```

Comment:
could you write it in a nicer way ? 
Like "The variable name is invalid, only alphanumerical characters and _ are accepted" ?